frontend-react/src/hooks/filters/mocks/MockFilterManager.ts

```diff
@@ -7,7 +7,7 @@ import { PageSettingsAction } from "../UsePages";
 
 const fakeDispatch = <T>(): Dispatch<T> => {
     return (_v: T) => {
-        console.log(_v);
+        // console.log(_v);
```

Comment:
Eh?

Reply:
The magic of main-merging

Reply:
Woops misread the diff. I commented out the console log to reduce test noise. Should this log stay?

Reply:
Let's remove the log call, but I feel like each of these dispatches could be replaced with a spy instead.  Not a huge deal either way -- hoping we can get off of FilterManager and CursorManager soon.